packages/core/zembleContext.ts

```diff
@@ -13,8 +12,7 @@ class ContextInstance implements Zemble.GlobalContext {
     T extends Zemble.KVPrefixes[K],
     K extends keyof Zemble.KVPrefixes = keyof Zemble.KVPrefixes
   >(prefix: K): IStandardKeyValueService<T> {
-    // @ts-expect-error fix sometime maybe :)
```

Comment:
Lesson: be mindful with @ts-expect-error :) @lordspace74 @Ryiguchi @RevanToma

KeyValue was not even imported.. ;)